libs/backend/src/load_ballot_image.ts

```diff
@@ -0,0 +1,25 @@
+import { loadImage, toDataUrl, toImageData } from '@votingworks/image-utils';
+
+/**
+ * Default factor by which to scale ballot images.
+ */
+export const CVR_BALLOT_IMAGE_SCALE = 0.5;
```

Comment:
If we shrink the images on export, does that mean we're also factoring this in to the layouts we use when cropping them in the adjudication flow? I'm assuming that must be what's going on but just want to check for my understanding

Reply:
I do not know, I just kept the image scale the same so it wouldn't mess anything up. Eventually we won't scale them at all, once we revisit how we're storing this images in VxAdmin.